community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/EntityUpdates.java

```diff
@@ -384,9 +414,9 @@ public int hashCode()
     {
         final int prime = 31;
         int result = 1;
-        result = prime * result + Arrays.hashCode( labelsBefore );
-        result = prime * result + Arrays.hashCode( labelsAfter );
-        result = prime * result + (int) (nodeId ^ (nodeId >>> 32));
+        result = prime * result + Arrays.hashCode( entityTokensBefore );
+        result = prime * result + Arrays.hashCode( entityTokensAfter );
+        result = prime * result + (int) (entityId ^ (entityId >>> 32));
```

Comment:
I see no real point in messing with generated hashcode and equals?

Reply:
At second glance:
1) it is not fully autogenerated
2) it is fundamentally broken since hash code value depends on iteration order
3) I see no reason of having equals/hashCode here at all; I couldn't find any corresponding usages